index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main class="container">
+        <header class="notification">
+            <div class="logo"><img src="img/My Logo.png" alt="Logo"></div>
+            <div class="hamburger"><nav><img src="img/Union.png" alt="menu"></nav></div>
+        </header>
+        <div class="main_body">
+            <div class="header2">I'm Ali<br> Glad to meet you!</div>
+            <div class="body2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
```

Comment:
Kindly use a `<p>` tag to wrap this paragraph instead of a `<div>`🙏. This is more semantically correct